src/components/ActivityCollector/utils/isPageViewEvent.js

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+export default event => {
+  const content = event.getContent();
+  return (
+    content.xdm &&
+    // NOTE: A page view event should "ideally" include the pageViews type
+    // && event.xdm.eventType === "web.webpagedetails.pageViews"
+    content.xdm.web &&
+    content.xdm.web.webPageDetails &&
+    content.xdm.web.webPageDetails.name
+  );
```

Comment:
Outside a lot of small details (like not looking at `XDM`) `AppMeasurement` operates the same way. `AppMeasurement` is limited to two types of requests, a page tracking event (`track`) and a link tracking event (`tl`). When a click doesn't qualify as a link click, the properties that we capture are just stored and submitted on a future call (for internal link clicks) which will need to be a page view if that data is to be kept. This is because a qualifying link click will erase and replace previously stored data. In the `Web SDK` we could argue that we should augment any future event with the data that we've captured previously but then we start to deviate further from how `AppMeasurement` manages click data (which might get reflected in reporting). However, this is something we might want to test in future releases given that the `Web SDK` is significantly more flexible in the type of events that can be submitted.